apps/ledger-live-mobile/src/screens/ReceiveFunds/02-AddAccountSelectDevice.tsx

```diff
@@ -90,11 +94,10 @@ export default function AddAccountsSelectDevice({
         onResult={onResult}
         onClose={onClose}
         request={{
-          currency: currency as CryptoCurrency,
-          // FIXME: IT SEEMS TokenCurrency WILL NEVER HAPPEN
-          // currency.type === "TokenCurrency"
-          //   ? currency.parentCurrency
-          //   : currency,
+          currency:
+            currency.type === "TokenCurrency"
```

Comment:
We should probably use `isTokenCurrency` here also if we can
```suggestion
            isTokenCurrency(currency)
```